Tests/IPAddressTests/IPAddressTests.cs

```diff
@@ -301,5 +301,24 @@ public void NetTest6_SocketAddressBasic()
                         + " as " + socketAddress2.ToString() + " " + socketAddress2.GetHashCode());
             }
         }
+
+        [TestMethod]
+        public void Equality_Tests()
+        {
+            var privateAddress = new IPAddress(new byte[] { 192, 168, 0, 1 });
+            var publicAddress = new IPAddress(new byte[] { 1, 1, 1, 1 });
+
+            // Equal
+            Assert.IsTrue(privateAddress.Equals(new IPAddress(new byte[]{192, 168, 0 , 1})), "192.168.0.1 equals 192.168.0.1");
```

Comment:
We should get away of "magic" numbers as they are prone to errors.
Better move these byte arrays for the addresses to readonly fields and/or having IP addresses as strings and then use Parse.

Reply:
Switch to `IPAddress.Parse`. Last I checked class level fields don't work because the test class isn't constructed (at least this was the case with the `SetupAttribute`.

Reply:
You're correct on that!
But static or const do work, right? (can't recall.... 😅 )